sympy/functions/special/hyper.py

```diff
@@ -220,6 +220,33 @@ def _eval_rewrite_as_Sum(self, ap, bq, z, **kwargs):
         return Piecewise((Sum(coeff * z**n / factorial(n), (n, 0, oo)),
                          self.convergence_statement), (self, True))
 
+    def _eval_nseries(self, x, n, logx):
+
+        from sympy.functions import factorial, RisingFactorial
+        from sympy import Order, Add
+
+        x0 = self.args[2].limit(x, 0)
+        ap = self.args[0]
+        bq = self.args[1]
+
+        if x0 != 0:
+            return super()._eval_nseries(x, n, logx)
```

Comment:
```suggestion
           return super()._eval_nseries(self, x, n, logx)
```

Reply:
I'm not sure of this, though. The docs say that if the second argument of `super` is omitted, then the method will be unbound but I don't know what will happen if both arguments are omitted. However, there is an example that seems to indicate that the returned method will be bound to `self`. Perhaps that should be tested.

Reply:
Tests seem to show that the method will be bound. So better leave this unchanged.

Reply:
I think what you are saying is the way it was done in python 2 and in python 3 both the arguments can be left omitted.